lib/mongoid/factory.rb

```diff
@@ -17,7 +17,7 @@ module Factory
     #
     # @return [ Document ] The instantiated document.
     def build(klass, attributes = nil)
-      type = (attributes || {})[TYPE]
+      type = (attributes || {}).with_indifferent_access[TYPE]
```

Comment:
Think it would be better to do:

```
attributes ||= {}
type = attributes[TYPE] || attributes[TYPE.to_sym]
```

Reply:
Yeah, sure!. It's done. Thanks

Reply:
@pacop Did you make the change suggested by @johnnyshields ?

Reply:
@estolfo Yeah, I forgot to push. Sorry, now it should be.

Reply:
@pacop Can you please correct the syntax error? There shouldn't be a space between the double ``|`` and the ``=``.

Reply:
@estolfo done, I don't know why it was there. Sorry again

Reply:
great, thank you